shared/teams/channel/header.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
+import * as Chat2Gen from '../../actions/chat2-gen'
+import {ConversationIDKey} from '../../constants/types/chat2'
+import {TeamID} from '../../constants/types/teams'
+import {pluralize} from '../../util/string'
+import {Activity} from '../common'
+
+type HeaderTitleProps = Kb.PropsWithOverlay<{
+  teamID: TeamID
+  conversationIDKey: ConversationIDKey
+}>
+
+const _HeaderTitle = (props: HeaderTitleProps) => {
+  const {teamID, conversationIDKey} = props
+  const {teamname} = Container.useSelector(s => Constants.getTeamMeta(s, teamID))
+  const {channelname, description, numParticipants} = Container.useSelector(
+    s =>
+      Constants.getChannelInfoFromConvID(s, teamID, conversationIDKey) ?? {
+        channelname: '',
+        description: '',
+        numParticipants: 0,
+      }
+  )
+  const yourOperations = Container.useSelector(s => Constants.getCanPerformByID(s, teamID))
+  const activityLevel = 'active' // TODO plumbing
+  const newMemberCount = 1 // TODO plumbing
+
+  const callbacks = useHeaderCallbacks(teamID, conversationIDKey)
+
+  const topDescriptors = (
```

Comment:
![image](https://user-images.githubusercontent.com/4698687/74879411-104cd980-5337-11ea-9ee3-ef40d8933a4f.png)
This tip is missing.